DEPLOYMENT.md

```diff
@@ -83,6 +88,10 @@ Or you could populate with fake data by running.
 go run ./util/cmd/load_fake_data/main.go -spanner_project=${SPANNER_PROJECT_ID} -spanner_instance=${SPANNER_INSTANCE_ID} -spanner_database=${SPANNER_DATABASE_ID} -datastore_project=${DATASTORE_PROJECT_ID} -datastore_database=${DATASTORE_DATABASE}
 ```
 
+Setup auth:
+
+Add your domain to the allow-list of domains in the [console](https://pantheon.corp.google.com/customer-identity/settings?project=webstatus-dev-internal-staging).
```

Comment:
Can this also be a console.cloud.google.com link rather than a corp one?

Reply:
Good idea! Done.